changelogs/unreleased/6566-lubronzhan-minor.md

```diff
@@ -0,0 +1,10 @@
+## Gateway API: handle Route conflicts with GRPCRoute.Matches
+
+It's possible that multiple GRPCRoutes will define the same Match conditions. In this case the following logic is applied to resolve the conflict:
+
+- The oldest Route based on creation timestamp. For example, a Route with a creation timestamp of “2020-09-08 01:02:03” is given precedence over a Route with a creation timestamp of “2020-09-08 01:02:04”.
+- The Route appearing first in alphabetical order (namespace/name) for example, foo/bar is given precedence over foo/baz.
+
+With above ordering, any GRPCRoute that ranks lower, will be marked with below conditions accordionly
```

Comment:
```suggestion
With above ordering, any GRPCRoute that ranks lower, will be marked with below conditions accordingly
```

Reply:
MM the codespell is not doing its work?

Reply:
Tried locally it doesn't work at all.

Reply:
hm yeah we might need to mess with configuration to make it run against changelogs if it isnt already

Reply:
It does. I tried using this binary locally against a file just contain `asdfasdf`, nothing happened 😢